in.bharatkalluri.splash.json

```
@@ -0,0 +1,36 @@
+{
+    "app-id" : "in.bharatkalluri.splash",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.34",
```

Update runtime to 3.36
```suggestion
    "runtime-version" : "3.36",
```